libraries/lib-import-export/ExportPlugin.h

```diff
@@ -39,6 +39,43 @@ struct IMPORT_EXPORT_API FormatInfo
    bool mCanMetaData;
 };
 
+
+class IMPORT_EXPORT_API ExportException
```

Comment:
It could be, but when I reviewed the implementation of `MessageBoxException`'s delayed handler I found out that this doesn't map well to the current behaviour, which I tried to preserve as much as possible. `ExportException` replaces (not literally ofcourse) `BasicUI::ShowMessageBox` which doesn't have counterpart in `AudacityException` hierarchy